src/app/artist-list/artist-list.component.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  SimpleChanges,
+  OnChanges
+} from '@angular/core';
+import { IArtist } from 'src/Interfaces/IArtist';
+import { ArtistService } from 'src/services/artist.service';
+import { take } from 'rxjs/operators';
+@Component({
+  selector: 'app-artist-list',
+  templateUrl: './artist-list.component.html',
+  styleUrls: ['./artist-list.component.css']
+})
+export class ArtistListComponent {
+  imageWidth = 50;
+  ImageMargin = 2;
+  errorMessage = '';
+  artists: IArtist[] = [];
+
+  searchstring: string;
```

Comment:
Please use camelCase for class members. i.e. `searchString`

Reply:
@strizzwald fixed